tests/Speckle.Sdk.Tests.Integration/Api/GraphQL/Resources/WorkspaceResourceTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Speckle.Sdk.Api;
+using Speckle.Sdk.Api.GraphQL.Resources;
+
+namespace Speckle.Sdk.Tests.Integration.API.GraphQL.Resources;
+
+public class WorkspaceResourceTests
+{
+  private readonly IClient _testUser;
+  private WorkspaceResource Sut => _testUser.Workspace;
+
+  public WorkspaceResourceTests()
+  {
+    var setupTask = Setup();
+    setupTask.Wait(); // Ensure setup runs synchronously for the constructor
```

Comment:
https://mderriey.com/2017/09/04/async-lifetime-with-xunit/

Reply:
Good suggestion, I will do this in a seperate PR to speed Oguzhan up, since he's blocked developing Grasshopper workspcaes support.

Reply:
Also, Chuck has suggested a way we can integration test the workspaces features properly by running a local server with workspaces enabled.
need to experiment a bit, but it should make it 100% test coverage